nes/src/bin/apu_demo.rs

```diff
@@ -22,7 +22,8 @@ fn main() {
   let device = audio_subsystem
     .open_playback(None, &desired_spec, |spec| {
       NesAudioProcess::new(recv, spec.freq as u32)
-    }).unwrap();
+    })
+    .unwrap();
```

Comment:
Is this the new format or the old one?

Reply:
Looks like everything is good (from travis builds passing)